internal/browser/browser.go

```diff
@@ -0,0 +1,376 @@
+package browser
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/muesli/termenv"
+
+	"github.com/hazelcast/hazelcast-commandline-client/internal/browser/layout/vertical"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/browser/multiline"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/termdbms/tuiutil"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/termdbms/viewer"
+)
+
+type StringResultMsg string
+type TableResultMsg *sql.Rows
+
+type controller struct {
+	tea.Model
+	client *hazelcast.Client
+}
+
+type table struct {
+	termdbmsTable viewer.TuiModel
+	keyboardFocus bool
+	lastIteration *SqlIterator
+}
+
+func (t *table) Init() tea.Cmd {
+	tuiutil.Faint = true
+	if lipgloss.ColorProfile() == termenv.Ascii {
+		tuiutil.Ascii = true
+		lipgloss.SetColorProfile(termenv.Ascii)
+	}
+	viewer.GlobalCommands["j"] = viewer.GlobalCommands["s"]
+	viewer.GlobalCommands["k"] = viewer.GlobalCommands["w"]
+	viewer.GlobalCommands["a"] = viewer.GlobalCommands["left"]
+	viewer.GlobalCommands["d"] = viewer.GlobalCommands["right"]
+	viewer.GlobalCommands["down"] = viewer.GlobalCommands["s"]
+	viewer.GlobalCommands["up"] = viewer.GlobalCommands["w"]
```

Comment:
@srknzl thanks for sharing this! A range check placed to avoid this panic on https://github.com/hazelcast/hazelcast-commandline-client/pull/65/commits/b91bc8bdb65f1e22766d04cb95a21678f6393bdd